src/preload.ts

```diff
@@ -22,7 +22,8 @@ function ipcRendererSend< T extends keyof IpcHandlers >(
 
 const api: IpcApi = {
 	archiveSite: ( id, format ) => ipcRendererInvoke( 'archiveSite', id, format ),
-	exportSiteToPush: ( id ) => ipcRendererInvoke( 'exportSiteToPush', id ),
+	exportSiteToPush: ( id, optionsToSync?, specificSelections? ) =>
```

Comment:
It's not a good practice to have optional parameters one by one, since it disallows providing the third argument w/o providing the second one. In our PHP codebase I saw a lot of such sad cases e.g. `fucntion(title, null, description)`. 
So I propose to combine them and wrap with parent object:
	
```suggestion
	exportSiteToPush: ( id, configuration ) =>
```

Reply:
I've made the adjustment.
Just FYI, that approach is not always the best, or should at least be used with caution.
On the frontend, for example, always using objects as arguments can lead to unwanted re-rendering due to equality issues. This doesn't apply to this context, so your suggestion is good. 👍